chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -49,8 +49,12 @@
       available = `df #{path} | tail -n 1 | sed "s/ \\+/ /g" | cut -d " " -f 4`
       db_size = `su - postgres -c 'pg_dumpall | wc -c'`
       if db_size.to_i > available.to_i
-        Chef::Log.fatal("DB size is: #{db_size}, available space on #{path} is #{available}")
-        raise "Not enough space on #{path} for the Database dump!"
+        message = "Not enough space for the Database dump on node #{node.name}!
+Database size is: #{db_size}B, available space on #{path} is only #{available}B."
+        Chef::Log.fatal(message)
+        node["crowbar_wall"]["chef-error"] = message
```

Comment:
The attribute should probably be `chef_error` and not `chef-error` (as we try to avoid dashes in attribute names).